test/controllers/users.js

```diff
@@ -49,7 +49,7 @@ describe('users controller', () => {
         })
         .catch((err) => {
           err.response.should.be.json;
-          err.response.body.error.message.should.equal('Validation error');
+          err.response.body.error.message.message.should.equal('Validation error');
```

Comment:
We should fix the code that is causing the doble nested message, instead of changing this should